http/dashboard_service.go

```diff
@@ -735,12 +734,40 @@ func (h *DashboardHandler) handlePostDashboardCell(w http.ResponseWriter, r *htt
 		EncodeError(ctx, err, w)
 		return
 	}
-	if err := h.DashboardService.AddDashboardCell(ctx, req.dashboardID, req.cell, req.opts); err != nil {
+	cell := new(platform.Cell)
+
+	opts := new(platform.AddDashboardCellOptions)
+	if req.UsingView != nil || req.Name != nil {
+		opts.View = new(platform.View)
+		if req.UsingView != nil {
+			// load the view
+			opts.View, err = h.DashboardService.GetDashboardCellView(ctx, req.dashboardID, *req.UsingView)
+			if err != nil {
+				EncodeError(ctx, err, w)
+				return
+			}
+		}
+		if req.Name != nil {
+			opts.View.Name = *req.Name
```

Comment:
Is the `Name` param supposed to describe the view name, or the dashboard name? The swagger says dashboard name.

Reply:
Should be the view's name, dashboard name is more like a parent thing like system dashboard. This name should be like cpu, memory, disk ...

Reply:
👍